cmd/av/stack_tree.go

```diff
@@ -26,32 +30,49 @@ var stackTreeCmd = &cobra.Command{
 			return err
 		}
 
+		currentBranch, err := repo.CurrentBranchName()
+		if err != nil {
+			return err
+		}
+
 		// TODO[polish]:
 		// 		We should show information about whether or not each branch is
 		//     	up-to-date with its stack parent as well as whether or not the
 		//		branch is up-to-date with its upstream tracking branch.
-		fmt.Println(defaultBranch)
+		if currentBranch == defaultBranch {
+			_, _ = fmt.Fprint(os.Stderr,
```

Comment:
We're using `fmt.Println` below which prints to stdout, not stderr, so let's be consistent here.

```suggestion
			_, _ = fmt.Print(
```